src/index.js

```diff
@@ -1,7 +1,7 @@
 import fetchApiData from './API/responseApiData';
 import displayBokomons from './Display/displayBokomons';
 
-import { counter } from './Display/counter';
+import {counter} from './Display/counter';
```

Comment:
- [ ] <b>REQUIRED:</b> This `import` is causing an error, Kindly address the issue and fix it.

![scrnli_1_8_2022_4-20-33 AM](https://user-images.githubusercontent.com/45217364/148628100-06a5df36-3853-449f-bd6f-8ac4ffa0a1e3.png)